src/sentry/features/manager.py

```diff
@@ -211,25 +211,29 @@ def has(
         >>> FeatureManager.has('organizations:feature', organization, actor=request.user)
 
         """
-        actor = kwargs.pop("actor", None)
-        feature = self.get(name, *args, **kwargs)
-
-        # Check registered feature handlers
-        rv = self._get_handler(feature, actor)
-        if rv is not None:
-            return rv
+        try:
+            actor = kwargs.pop("actor", None)
+            feature = self.get(name, *args, **kwargs)
 
-        if self._entity_handler and not skip_entity:
-            rv = self._entity_handler.has(feature, actor)
+            # Check registered feature handlers
+            rv = self._get_handler(feature, actor)
             if rv is not None:
                 return rv
 
-        rv = settings.SENTRY_FEATURES.get(feature.name, False)
-        if rv is not None:
-            return rv
+            if self._entity_handler and not skip_entity:
+                rv = self._entity_handler.has(feature, actor)
+                if rv is not None:
+                    return rv
+
+            rv = settings.SENTRY_FEATURES.get(feature.name, False)
+            if rv is not None:
+                return rv
 
-        # Features are by default disabled if no plugin or default enables them
-        return False
+            # Features are by default disabled if no plugin or default enables them
+            return False
+        except Exception as e:
+            sentry_sdk.capture_exception(e)
```

Comment:
I'd recommend using `logging.exception("Failed to run feature check")` instead. That way you'll still capture the exception, but can provide a more specific error title

Reply:
sure, done 👍